latch_cli/centromere/ctx.py

```diff
@@ -298,10 +302,48 @@ def __init__(
                 )
             self.version = self.version.strip()
 
-            if not self.disable_auto_version:
-                hash = hash_directory(self.pkg_root)
-                self.version = f"{self.version}-{hash[:6]}"
-                click.echo(f"  {self.version}\n")
+            from git import GitError, Repo
```

Comment:
add to setup.py + move import to top level unless its a really hefty import

Reply:
The import will fail if git is not installed locally, so I am going to import here and catch ImportError